scripts/apps/master-desk/components/AssignmentsComponent.tsx

```diff
@@ -70,7 +70,7 @@ export class AssignmentsComponent extends React.Component<IProps, IState> {
 
         if (this.state.deskData && this.state.deskData[desk._id]) {
             this.state.deskData[desk._id].forEach((item) =>
-                this.assignmentStages.some((stage) =>
+                this.assignmentStages.find((stage) =>
```

Comment:
Can you also simplify it and split it in 2 steps? First find the assignment stage(without modifying anything) and then use that stage to increment the total.